ftl_installer/parser/Parser.py

```diff
@@ -37,6 +50,12 @@ def main():  # pragma: no cover
     ftl_logger.addHandler(ftl_stream_handler)
     ftl_logger.debug("initialized ftl-logging at level: {0}".format(log_level))
 
+    # If the configuration file is provided, parse and store it.
+    if args.config:
+        ftl_config = ConfigHandler(args.config)
+        # for test purpose, remove later
+        pprint.pprint(ftl_config.stored_config)
```

Comment:
If using a method to convert the config object into a yaml string, then print can be used here instead of importing and using pprint.